Sources/ProcedureKit/Collection+ProcedureKit.swift

```diff
@@ -9,16 +9,18 @@ import Foundation
 extension Collection where Iterator.Element: Operation {
 
     internal var operationsAndProcedures: ([Operation], [Procedure]) {
-        return reduce(([], [])) { result, element in
-            var (operations, procedures) = result
+        // this for-in loop has considerably better performance characteristics than reduce()
```

Comment:
👍 

If anyone's curious, see https://airspeedvelocity.net/2015/08/03/arrays-linked-lists-and-performance/ for more info.